frontend/src/lib/lemon-ui/LemonButton/LemonButton.scss

```diff
@@ -4,7 +4,6 @@
     position: relative;
     display: flex;
     flex-direction: row;
-    flex-shrink: 0;
```

Comment:
Looks like this might be causing some snapshot issues

Reply:
Yeah I wanted the PR to see what would be affected. The issue is if we have that property here its pretty much impossible to override nicely...